catalog/nsili/catalog-nsili-transformer/src/test/java/com/connexta/alliance/nsili/transformer/TestDAGConverter.java

```diff
@@ -2647,4 +2701,77 @@ private static Calendar getDefaultCalendar() {
         int second = 10;
         return new GregorianCalendar(year, month, dayOfMonth, hourOfDay, minute, second);
     }
+
+    private void setupMocks() throws Exception {
+        ResourceResponse testReponse = new ResourceResponse() {
```

Comment:
I think you could mock Resource and ResouceResponse instead of implementing them here